.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
   - 0.10
+  - 4
```

Comment:
Thanks, you’re right – didn’t think of this as the previous entry didn’t use them either. :)